### ISSUE TITLE
Preferences window does not close on Escale or Ctrl+W

### ISSUE THREAD

**Issue by exander77** [tkambler/electron-preferences#130]
I have this code as part of my configuration, I would assume it allows the window to be closed by `Ctrl+W`:
```
  menu: Menu.buildFromTemplate(
    [
      {
        label: 'Window',
        role: 'window',
        submenu: [
          {
            label: 'Close',
            accelerator: 'CmdOrCtrl+W',
            role: 'close'
          }
        ]
      }
    ]
  ),
```

**Comment by pvrobays** [comment]
Hi @exander77 
I do not know why it is not working. Should investigate further. However doe now you could implement your own click handler and close the preference window via the exposed close function. I hope this helps

**Comment by exander77** [comment]
@pvrobays Sadly preferences object does not expose close function, only show.

**Comment by pvrobays** [comment]
Hi @exander77. Meanwhile I've had the time to have a quick look.
The code example you've mentioned is copied from the README but has a bug in it. The property name shouldn't be `menu` but `menuBar`. Once you change that, the menu option works and you'll be able to close the preference window with the shortcut as you mentioned. Let me know if this works for you.
Also the close method isn't implemented yet, thought it already was, woops :)

I'll leave this ticket open to fix the two things:
1. fix README to use `menuBar`
2. implement close method on preferences object

**Comment by pvrobays** [comment]
FYI fixed in development. Will be in the next release

**Comment by pvrobays** [comment]
Fix released in 2.6.0
